scalardb-analytics-spark-sample/README.md

```diff
@@ -0,0 +1,5 @@
+# Sample Application with ScalarDB Analytics with Spark
+
+This is a sample application that demonstrates how you can run analytical queries on the data managed by [ScalarDB](https://www.scalar-labs.com/scalardb/) using [ScalarDB Analytics with Spark](https://scalardb.scalar-labs.com/docs/latest/scalardb-analytics-spark/getting-started/).
+
+Please refer to [the documentation](https://scalardb.scalar-labs.com/docs/latest/scalardb-samples/scalardb-analytics-spark-sample/) for more information.
```

Comment:
Just to confirm, will the doc mentioned here be created and added to the sample tutorials in the centralized docs repository?

Reply:
Exactly. I'll create another PR to `docs-internal-scalardb` soon that adds user documentation of ScalarDB Analytics with Spark, including a doc about the sample app as well.

Reply:
Understood. Thank you!🙏